app/controllers/api/v4/application_controller.rb

```diff
@@ -36,9 +36,7 @@ def build_link_headers(links)
   end
 
   def force_ssl
-    if !request.ssl?
-      render status: 301, json: {error: "The Splits I/O API is only accessible over HTTPS."}
-    end
+    render status: 301, json: {error: 'The Splits I/O API is only accessible over HTTPS.'} unless request.ssl?
```

Comment:
This was my original goof in v3 that I'm just now noticing, but if this is a 301 it should have a `Location: https://the-url-you-were-trying-to-hit` header set. Or if that's hard to do, just a 400 would do fine.

Reply:
I'll take a look into it.  Do you also what to have a `location` key in the returned json response?

Reply:
Nah just the header is fine.